testAlarms.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# # Loop for Command 1
+# for i in {1..50}; do
```

Comment:
Do we need to keep all these commented code?

Reply:
Removed